CHANGELOG.md

```diff
@@ -13,8 +13,243 @@ _Note: Gaps between patch versions are faulty, broken or test releases._
 
 See [CHANGELOG - v4](https://github.com/webdriverio-boneyard/v4/blob/master/CHANGELOG.md).
 
-## v5.0.0 (2018-12-12)
+---
+
+## v5.0.0 (2018-tbd)
 
 #### :boom: Breaking Change
-* `@wdio/cli`
-    * contains now the wdio cli command
+* moved `wdio` cli command from [`webdriverio`](https://www.npmjs.com/package/webdriverio) package to [`@wdio/cli`](https://www.npmjs.com/package/@wdio/cli)
+* commands are scoped to `browser` and `element` - no selector property on commands anymore
+  ```js
+  // v4
+  browser.click('#myElem')
+  ```
+  ```js
+  // v5 and upwards
+  const elem = $('#myElem')
+  elem.click()
+  ```
+* no command chaining anymore (in standalone and wdio mode)
+  ```js
+  // v4 (standalone/async mode)
+  browser
+    .url('https://duckduckgo.com/')
+    .setValue('#search_form_input_homepage', 'WebdriverIO')
+    // ...
+  ```
+  ```js
+  // v5
+  await browser.url('https://duckduckgo.com/')
+  const elem = await browser.$('#search_form_input_homepage')
+  await elem.click()
+  ```
+* every protocol command returns `value` property instead of raw driver response
```

Comment:
* every protocol command returns a `value` property instead of raw driver response